lib/deppack.js

```diff
@@ -183,9 +191,13 @@ const xBrowserResolve = (mod, opts, cb) => {
         err = isGlob ? `Could not load global module '${mod}'.` : `Could not load module '${mod}' from '${src}'.`;
 
         if (packages[topLevel] == null) {
-          err = err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
+          err += ` Possible solution: add '${topLevel}' to package.json and \`npm install\`.`;
         } else {
-          err = err += ` Possible solution: run \`npm install\`.`;
+          if (overrides[mod]) {
+            err += ` Possible solution: check your overrides in package.json.`;
```

Comment:
Having something in overrides shouldn't automatically rule out the `npm install` tip.